<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="10" passed="9">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="21620" started-at="2019-03-23T21:54:59Z" finished-at="2019-03-23T21:55:20Z">
    <groups>
    </groups>
    <test name="Test Maven Suite" duration-ms="21620" started-at="2019-03-23T21:54:59Z" finished-at="2019-03-23T21:55:20Z">
      <class name="com.qa.tests.Log4jDemo">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.tests.Log4jDemo@51efea79]" name="setUp" is-config="true" duration-ms="9738" started-at="2019-03-23T17:54:59Z" finished-at="2019-03-23T17:55:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="validateTitle()[pri:1, instance:com.qa.tests.Log4jDemo@51efea79]" name="validateTitle" duration-ms="303" started-at="2019-03-23T17:55:08Z" finished-at="2019-03-23T17:55:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateTitle -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.qa.tests.Log4jDemo@51efea79]" name="quit" is-config="true" duration-ms="724" started-at="2019-03-23T17:55:09Z" finished-at="2019-03-23T17:55:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.tests.Log4jDemo@51efea79]" name="setUp" is-config="true" duration-ms="5651" started-at="2019-03-23T17:55:09Z" finished-at="2019-03-23T17:55:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="LoginClick()[pri:2, instance:com.qa.tests.Log4jDemo@51efea79]" name="LoginClick" duration-ms="4182" started-at="2019-03-23T17:55:15Z" finished-at="2019-03-23T17:55:19Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Cogmento CRM] but found [CRM]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Cogmento CRM] but found [CRM]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at org.testng.Assert.assertEquals(Assert.java:188)
	at com.qa.tests.Log4jDemo.LoginClick(Log4jDemo.java:72)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginClick -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.qa.tests.Log4jDemo@51efea79]" name="quit" is-config="true" duration-ms="743" started-at="2019-03-23T17:55:19Z" finished-at="2019-03-23T17:55:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- com.qa.tests.Log4jDemo -->
      <class name="com.qa.tests.DemoTest1">
        <test-method status="PASS" signature="diff()[pri:0, instance:com.qa.tests.DemoTest1@36d4b5c]" name="diff" duration-ms="12" started-at="2019-03-23T17:54:59Z" finished-at="2019-03-23T17:54:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- diff -->
        <test-method status="PASS" signature="div()[pri:0, instance:com.qa.tests.DemoTest1@36d4b5c]" name="div" duration-ms="0" started-at="2019-03-23T17:54:59Z" finished-at="2019-03-23T17:54:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- div -->
        <test-method status="PASS" signature="mult()[pri:0, instance:com.qa.tests.DemoTest1@36d4b5c]" name="mult" duration-ms="0" started-at="2019-03-23T17:54:59Z" finished-at="2019-03-23T17:54:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mult -->
        <test-method status="PASS" signature="sum()[pri:0, instance:com.qa.tests.DemoTest1@36d4b5c]" name="sum" duration-ms="0" started-at="2019-03-23T17:54:59Z" finished-at="2019-03-23T17:54:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sum -->
      </class> <!-- com.qa.tests.DemoTest1 -->
      <class name="com.qa.tests.DemoTestTestNGclass">
        <test-method status="PASS" signature="diff()[pri:0, instance:com.qa.tests.DemoTestTestNGclass@6d00a15d]" name="diff" duration-ms="1" started-at="2019-03-23T17:54:59Z" finished-at="2019-03-23T17:54:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- diff -->
        <test-method status="PASS" signature="mult()[pri:0, instance:com.qa.tests.DemoTestTestNGclass@6d00a15d]" name="mult" duration-ms="1" started-at="2019-03-23T17:54:59Z" finished-at="2019-03-23T17:54:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mult -->
        <test-method status="PASS" signature="div()[pri:0, instance:com.qa.tests.DemoTestTestNGclass@6d00a15d]" name="div" duration-ms="0" started-at="2019-03-23T17:54:59Z" finished-at="2019-03-23T17:54:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- div -->
        <test-method status="PASS" signature="sum()[pri:0, instance:com.qa.tests.DemoTestTestNGclass@6d00a15d]" name="sum" duration-ms="0" started-at="2019-03-23T17:54:59Z" finished-at="2019-03-23T17:54:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sum -->
      </class> <!-- com.qa.tests.DemoTestTestNGclass -->
    </test> <!-- Test Maven Suite -->
  </suite> <!-- Suite -->
</testng-results>
